src/main/util.ts

```diff
@@ -581,6 +582,69 @@ const isNumberClose = (val: number, compare: number) => {
     return Math.abs(compare - val) <= 2;
 };
 
+/**
+ * Get a text string that indicates the physical position of a display depending
+ * on its index.
+ */
+const getDisplayPhysicalPosition = (count: number, index: number): string => {
+    if (index === 0)         return 'Left';
+    if (index === count - 1) return 'Right';
+
+    return 'Middle #' + index;
+};
+
+/**
+ * Get and return a list of available displays on the system sorted by their
+ * physical position.
+ *
+ * This makes no attempts at being perfect - it completely ignores the `bounds.y`
+ * property for people who might have stacked their displays vertically rather than
+ * horizontally. This is okay.
+ */
+const getAvailableDisplays = (): OurDisplayType[] => {
+    const primaryDisplay = screen.getPrimaryDisplay();
+    const allDisplays = screen.getAllDisplays();
+
+    // Create an unsorted list of Display IDs to zero based monitor index
+    // So we're can use that index later, after sorting the displays according
+    // to their physical location.
+    const displayIdToIndex: {[key: number]: number } = {}
+    allDisplays.forEach((display: Display, index: number) => displayIdToIndex[display.id] = index)
+
+    // Iterate over all available displays and make our own list with the
+    // relevant attributes and some extra stuff to make it easier for the
+    // frontend.
+    const ourDisplays: OurDisplayType[] = [];
+    const numberOfMonitors = allDisplays.length;
+
+    allDisplays
+        .sort((A: Display, B: Display) => A.bounds.x - B.bounds.x)
+        .forEach((display: Display, index: number) => {
+            const isPrimary = display.id === primaryDisplay.id;
+            const displayIndex = displayIdToIndex[display.id];
+            const { width, height } = display.size;
+            console.log(display)
+
+            ourDisplays.push({
+                id: display.id,
+                index: displayIndex,
+                physicalPosition: getDisplayPhysicalPosition(numberOfMonitors, index),
+                primary: isPrimary,
+                displayFrequency: display.displayFrequency,
+                depthPerComponent: display.depthPerComponent,
+                size: display.size,
+                scaleFactor: display.scaleFactor,
+                aspectRatio: width / height,
+                physicalSize: {
```

Comment:
This makes me a bit nervous. How confident are you that we're not regressing recent fixes in this area to people who are unlucky with the rounding of their resolutions?

Reply:
Because this is the correct way to do this. 2560.5 (like mine) becomes 2560, which is accurate. I'd be happy to make a test of various resolutions and scaling factors and apply the logic to it with a table to show the results.

Reply:
What makes floor the correct thing rather than ceil? 

What if someone someone's rounding gives then 2559.5 which becomes 2559?

Reply:
Because of the examples I've seen, it'll _never_ be lower than the actual resolution - however, I'm working on a fix that - instead of checking if a number "is close", it will simply find the closest resolution to whatever is given. That'll sort it out once and for all - and I can test it because the user who reported this issue posted his display settings.

Reply:
Also, I didn't remove the `isNumberClose()` logic, which is "the heart" of this - but really, I can remove `Math.floor()` - it doesn't really matter _to me_.

Reply:
Please see #154 before considering this.

Reply:
Regression crisis averted! Since #154 was merged, you may re-consider this PR.